php/async/bytetrade.php

```diff
@@ -447,7 +447,7 @@ public function fetch_order_book($symbol, $limit = null, $params = array ()) {
             $request['limit'] = $limit; // default = maximum = 100
         }
         $response = yield $this->marketGetDepth (array_merge($request, $params));
-        $timestamp = $this->safe_integer($response, 'timestamp');
+        $timestamp = $this->safe_value($response, 'timestamp');
```

Comment:
these seem to be remnants from other PRs.

Reply:
@ttodua that generated file is outdated, we just don't/should commit them every time